src/Avalonia.Visuals/Media/DrawingContext.cs

```diff
@@ -59,7 +59,7 @@ private set
         //HACK: This is a temporary hack that is used in the render loop 
         //to update TransformedBounds property
         [Obsolete("HACK for render loop, don't use")]
-        internal Matrix CurrentContainerTransform => _currentContainerTransform;
+        public Matrix CurrentContainerTransform => _currentContainerTransform;
```

Comment:
Yeah... How to properly handle this is something we've been debating for like years... I'm fine with this hack for now personally, but we need to work out how we're going to do this properly.